src/js/pages/JobsPage.js

```diff
@@ -2,7 +2,7 @@ import mixin from 'reactjs-mixin';
 import React from 'react';
 import {RouteHandler} from 'react-router';
 
-import IconJobs from 'babel!svg-react!../../img/components/icons/pages.svg?name=IconJobs';
+import IconJobs from 'babel!svg-react!../../img/components/icons/pages-code.svg?name=IconJobs';
```

Comment:
Are we going to load all SVGs like this? If so, maybe we should define the loaders in our webpack config so we don't have to prefix with `babel!svg-react!` here. What do you think?